README.md

```diff
@@ -64,24 +64,19 @@ Try it on [her website!](https://olivia-ai.org)
 ### Installation 
 #### Login to Github 
 
-To get a personal accsess token from Github go to Setings>Developer settings>Personal access tokens
+`To get a personal access token from Github go to Setings > Developer settings > Personal Access Tokens`
```

Comment:
I'd just put Setings > Developer settings > Personal Access Tokens inside

Reply:
Ok I see what you meant now, sorry for that.